src/identities/dto/rotate-primary-key-params.dto.ts

```diff
@@ -0,0 +1,24 @@
+/* istanbul ignore file */
+
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { IsDate, IsOptional, IsString } from 'class-validator';
+
+import { TransactionBaseDto } from '~/common/dto/transaction-base-dto';
+
+export class RotatePrimaryKeyParamsDto extends TransactionBaseDto {
+  @ApiProperty({
+    description: 'Account address to which to rotate the primary key',
+    example: '5grwXxxXxxXxxXxxXxxXxxXxxXxxXxxXxxXxxXxxXxxXxxXx',
+  })
+  @IsString()
+  readonly targetAccount: string;
+
+  @ApiPropertyOptional({
+    description: 'Date at which the Identity will expire',
+    example: new Date(new Date().getTime() + +365 * 24 * 60 * 60 * 1000).toISOString(),
```

Comment:
```suggestion
    example: new Date(new Date().getTime() + 365 * 24 * 60 * 60 * 1000).toISOString(),
```
Does the number side need the unary number cast? looks more like left over since its already a number